vocode/streaming/streaming_conversation.py

```diff
@@ -1272,14 +1279,22 @@ async def send_speech_to_output(
             self.logger.debug("Interrupted speech output on the last chunk")
             if not moved_back:
                 self.agent.move_back_state()
-            return "", False
-
+            message_sent = synthesis_result.get_message_up_to(
+                time.time() - speech_start
+            )
+            self.log_agent_response(BaseMessage(text=message_sent))
+            if json_transcript_entry:
+                json_transcript_entry.message_sent = message_sent
+            return message_sent, True
         self.transcriptions_worker.synthesis_done = True
-
         if self.transcriptions_worker.buffer_check_task:
-            return "", False
```

Comment:
we're changing the return a bit here; what do these values mean?

Reply:
One is the message sent. The other is a flag for if the speech was interrupted. Both are unused as far as i know